Mobile version

luccamicale opened the pull request:
I created mi first capstone. I complete all the projecy using HTML CSS and javascript

luccamicale committed:
initial commit

luccamicale committed:
fix linters

luccamicale committed:
I put the media queris

luccamicale committed:
I finished with my Capstone

luccamicale committed:
I changed one things in my project

levy002 commented on about.html:
```diff
@@ -0,0 +1,129 @@
+                <li id="english"><a class="nav-inside" href="#"></a>English</li>
+                <li id="menu"><a class="menuu" href="#"><img src="images+/Icon-Menu.jpg" alt="main-menu"></a></li>
+                <a href="index.html" ><img id="logo" src="./images+/logo.jpg" alt="Logo"></a>
+            </ul>
```
- [ ] Kindly make sure to wrap your top navigation bar into the `header` tag.
- [ ] I highly recommend you wrap the top bar into the `nav` tag.

levy002 commented on about.html:
```diff
@@ -0,0 +1,129 @@
+                <li id="twitter"><img class="nav-inside" src="images+/gorjeo.png" alt=""></li>
+                <li id="login"><a class="nav-inside" href="#"></a>Login</li>
+                <li id="mypage"><a class="nav-inside" href="#"></a>MyPage</li>
+                <li id="english"><a class="nav-inside" href="#"></a>English</li>
```
- [ ] I highly recommend adding the text to the `a` tag instead of the `li` tag. :wink: 
   
     - ### Example
   `<li><a>Login</a><li>`

levy002 commented on index.html:
```diff
@@ -0,0 +1,188 @@
+                <li id="english"><a  class="nav-inside" href="#"></a>English</li>
+                <li id="menu"><a class="menuu" href="#"><img src="images+/Icon-Menu.jpg" alt="main-menu"></a></li>
+                <li id="home"><p class="homee">Home Decoration</p></li>
+            </ul>
```
- [ ] Kindly make sure to wrap your top navigation bar into the `header` tag.
- [ ] I highly recommend you wrap the top bar into the `nav` tag.

levy002 commented on index.html:
```diff
@@ -0,0 +1,188 @@
+                <li id="mypage"><a  class="nav-inside"href="#"></a>MyPage</li>
+                <li id="english"><a  class="nav-inside" href="#"></a>English</li>
+                <li id="menu"><a class="menuu" href="#"><img src="images+/Icon-Menu.jpg" alt="main-menu"></a></li>
+                <li id="home"><p class="homee">Home Decoration</p></li>
```
- [ ] I highly recommend adding the text to the `a` tag instead of the `li` tag. :wink: 
   
     - ### Example
   `<li><a>Login</a><li>`

levy002 commented on style.css:
```diff
@@ -0,0 +1,669 @@
+  display: flex;
+  flex-direction: row;
+}
+
```
- [ ] Kindly make sure to align correctly your cards, I noticed that it is bigger than the actual width of the page on the mobile version.
 
     - ### Your design on mobile version (width: 375px)
   ![image](https://user-images.githubusercontent.com/87197412/191818620-b949e44a-bcdc-42a3-83a9-85df5ec735a3.png)

- [ ] Kindly make sure to add the `JOIN CAMPAIN` button in order to meet the provided design.

  - ### Behance design
![image](https://user-images.githubusercontent.com/87197412/191820344-7ee59fa4-46f1-4232-97c7-7f6dddf29210.png)

levy002 commented on style.css:
```diff
@@ -0,0 +1,669 @@
+  width: 60px;
+}
+
+#speakers {
+  margin-left: 45px;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
```
- [ ] KIndly adjust your margin in order to align the speakers correctly as the provided design.

  - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/191819730-8d7ab947-c6ab-4ba7-ad21-eec3d5b60dd9.png)


  - ### Provided design
![image](https://user-images.githubusercontent.com/87197412/191819834-8bbb4f66-344f-4d7b-bbcc-4452735bc2d8.png)

levy002 commented on index.html:
```diff
@@ -0,0 +1,188 @@
+                <a id="button-more" href="#speakers">More</a>
+            </div>
+        </div>
+        <div id="container-partners">
+            <div id="partner-title"><h3 id="title-second">Social Media and Partners</h3></div>
+            <hr class="line5">
+            <div class="container-ul-partners">
+                <div id="partner-objects">
+                    <ul id="classes-photos-partners">
+                        <li><img class="image-partners" src="images+/gorjeo (1).png" alt=""></li>
+                        <br>
+                        <li><img class="image-partners" src="images+/tik-tok.png" alt=""></li>
+                        <br>
+                        <li><img class="image-partners" src="images+/facebook(1).png" alt=""></li>
+                    </ul>
+                </div>
+                <div id="partner-objects-two">
+                    <ul id="classes-photos-partners-two">
+                        <br>
+                        <li><img class="image-partners" id="image-partners-one" src="images+/instagram.png" alt=""></li>
+                        <br>
+                        <li><img class="image-partners" id="image-partners-two" src="images+/whatsapp.png" alt=""></li>
+                        <br>
+                        <li><img class="image-partners" src="images+/youtube.png" alt=""></li>
+                    </ul>
+                </div>
+            </div>
```
- [ ] Kindly make sure to hide the partner's container on home page page of mobile version in order to stick to the provided design.

   - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/191821467-0f587a98-0f15-4b9f-b73d-6e5dc21076d4.png)


   - ### Behance design
   ![image](https://user-images.githubusercontent.com/87197412/191821407-58c7250e-a4c2-42c8-ad4e-5896fb17a1c4.png)

levy002 commented on about.html:
```diff
@@ -0,0 +1,129 @@
+            </nav>
+          </header>
+        
+        <div id="main-section">
+            <div>
+                <h2 id="sub-title">Hello!!! Introducing to the world</h2>
+                <h1 id="title">Home Decoration Page 2022</h1>
+                <h3 id="sub-title-2">In this section about us, we will tell you how our launch to the market was and how we were able to locate ourselves</h3>
+                <br>
+            </div>
```
- [ ] Kindly make sure to adjust the title styling in order to meet the provided design.

   - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/191822420-4750b6ae-dfad-4167-b3f3-7700c0d7b8a0.png)

  - ### Behance design

   ![image](https://user-images.githubusercontent.com/87197412/191822307-1b99990b-b803-45fc-96d1-3b544bccf434.png)

levy002 reviewed (changes requested):
## Hello @luccamicale   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif)

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to remove unnecessary content on your mobile navigation bar in order to meet the provided design. Kindly the same background color as the Behance design.

   - ### Your design
  ![image](https://user-images.githubusercontent.com/87197412/191823369-190ef30a-37b9-41e8-97b5-68aa38637a46.png)

![image](https://user-images.githubusercontent.com/87197412/191823657-69ae6188-6931-42ca-956e-d80f2fc39180.png)



   - ### Behance design
   ![image](https://user-images.githubusercontent.com/87197412/191823120-bb8f5c9a-17eb-4a80-98e6-54bc82a535a4.png)

- [ ] I highly recommend aligning all partners correctly in order to meet the provided design.

   - ### Your design

    ![image](https://user-images.githubusercontent.com/87197412/191824413-28c32acf-059a-4f32-a216-3d57ecf1e2da.png)


  - ### Behance design
    ![image](https://user-images.githubusercontent.com/87197412/191824322-4251ddb6-4c7e-4fc6-b36b-c5d4829b3e9d.png)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luccamicale committed:
I modify only the parts of home

luccamicale committed:
more changes

luccamicale committed:
I finished with the changes in my project

luccamicale committed:
I added more changes

luccamicale committed:
I fix the errors

luccamicale commented:
Hi! I changed all things bad in my project. 
![1](https://user-images.githubusercontent.com/107897361/191869033-0532ea4b-cd7b-42b8-a0ef-3d57261eddcd.PNG)
![5](https://user-images.githubusercontent.com/107897361/191869059-f208cfa7-c0d4-4601-bbd6-f3ac91b3c756.PNG)
![4](https://user-images.githubusercontent.com/107897361/191869062-c7f135b7-5731-47db-a64c-551ce7caceb9.PNG)
![3](https://user-images.githubusercontent.com/107897361/191869063-a60658d1-0228-4aef-afaf-81f71d5d3263.PNG)
![2](https://user-images.githubusercontent.com/107897361/191869065-3e28c48f-4733-44b6-8abc-a76a1c9b7d45.PNG)

Please pay a lot atention! I fixed all things

Ugboaja-Uchechi commented on style.css:
```diff
@@ -0,0 +1,698 @@
+  width: 50px;
+  margin-left: 15px;
+}
+
+#container-partners {
+  background-color: rgb(86, 90, 90);
+  margin-bottom: 20px;
+  margin-top: 10px;
+}
```
- [x] Great job working on the requested changes 👏 . Unfortunately, one was missed. As the previous reviewer mentioned kindly remove the `partner section` in the `mobile version` `home page` 🙏 

![image](https://user-images.githubusercontent.com/74814780/191872123-95432146-ae94-41af-be14-127ad6add1f2.png)

Ugboaja-Uchechi reviewed (changes requested):
Hi @luccamicale, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luccamicale committed:
I fix one change in my project

luccamicale commented:
I changed the one thing bad in the project. Now all it is good!!!
![6](https://user-images.githubusercontent.com/107897361/191875802-e978d0e3-952b-4f5a-a690-1c0017be6a40.PNG)

franclobo reviewed (approved):
Hi, @luccamicale, 

#STATUS: APPROVED 👏🏽 👏🏽 👏🏽 
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Good communication with code reviewers 💯 


**You could consider:**
- [ ] **[OPTIONAL]** To fix the LICENSE link in your README file we could use `[MIT](./MIT.md)` instead of `[MIT](./LICENSE)` it depends of the file's name.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
